config/locales/es/general.yml

```diff
@@ -242,7 +242,7 @@ es:
       my_account_link: Mi cuenta
       my_activity_link: Mi contenido
       open: abierto
-      open_gov: Gobierno %{open}
+      open_gov: Gobierno abierto
```

Comment:
Nice catch!

Reply:
Done!

Reply:
I've just noticed the `eu` locale still uses the `%{open}` parameter :sweat_smile:. Would it be possible to change that file in crowdin and update this pull request to include it? Otherwise the page using this translation will crash :thinking:.

Reply:
If the previous line is right, I guess the right translation would be `open_gov: Gobernu irekia` :thinking:.